src/systems/ec/implementations/react/templates/pages/person-list/src/PersonListPage.jsx

```diff
@@ -0,0 +1,527 @@
+                  <DescriptionTerm>Phone number</DescriptionTerm>
+                  <DescriptionDefinition>+32 2 29 64816</DescriptionDefinition>
+                  <DescriptionTerm>Responsibilities</DescriptionTerm>
+                  <DescriptionDefinition>
+                    <UnorderedList className="ecl-u-mv-m">
+                      <UnorderedListItem>
+                        Food information and composition
+                      </UnorderedListItem>
+                      <UnorderedListItem>
+                        Food processing technologies and novel foods
+                      </UnorderedListItem>
+                    </UnorderedList>
+                    <Link
+                      variant="standalone"
+                      label="See all responsibilities"
+                      href="/example"
+                      icon={{
+                        shape: 'ui--corner-arrow',
+                        size: 'xs',
+                        transform: 'rotate-90',
+                      }}
+                    />
+                  </DescriptionDefinition>
+                </DescriptionList>
+              </div>
+            </article>
+
+            <article className="ecl-u-d-flex ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-pa-s ecl-u-mb-xl ecl-u-mb-lg-3xl">
+              <div
+                role="img"
+                aria-label="Example image"
+                className="ecl-u-flex-shrink-0 ecl-u-mr-l ecl-u-media-a-m"
+                style={{
+                  backgroundImage:
+                    'url("https://upload.wikimedia.org/wikipedia/commons/thumb/2/27/Emojione_1F464.svg/1024px-Emojione_1F464.svg.png")',
+                  backgroundSize: 'contain',
+                }}
+              />
+              <div className="ecl-u-flex-grow-1">
+                <h3 className="ecl-u-type-heading-3 ecl-u-type-prolonged-m ecl-u-type-bold ecl-u-mt-none ecl-u-mb-xs">
+                  Natasha Bertaud
+                </h3>
+                <p className="ecl-u-type-paragraph ecl-u-type-uppercase ecl-u-type-color-grey-75 ecl-u-mt-none ecl-u-type-s">
+                  Deputy Chief Spokesperson
+                </p>
+                <DescriptionList variant="horizontal">
+                  <DescriptionTerm>Phone number</DescriptionTerm>
+                  <DescriptionDefinition>+32 2 29 64816</DescriptionDefinition>
+                  <DescriptionTerm>Email</DescriptionTerm>
+                  <DescriptionDefinition>
+                    <a href="mailto:natasha.bertaud@ec.europa.eu">
+                      natasha.bertaud@ec.europa.eu
+                    </a>
+                  </DescriptionDefinition>
+                  <DescriptionTerm>Social media</DescriptionTerm>
+                  <DescriptionDefinition className="ecl-u-mv-none ecl-u-pv-none">
+                    <SocialMediaFollow
+                      className="ecl-u-mt-m ecl-u-pa-none ecl-u-bg-white"
+                      links={[
```

Comment:
For all social links (Twitter, Facebook, ..) margin-right to the icon seems too wide. Spacing between the icon and label is too large. 

```
.ecl-social-media-follow__icon, .ecl-social-media-follow__icon-hover {
    flex-shrink: 0;
    margin-right: 1rem;
    top: 0!important;
}
```
I think a margin-right of `.5rem` would be better

Reply:
I'm using a predefined component and the rules follow